src/main/resources/templates/userprofile.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta charset="UTF-8">
+    <title>You are logged in</title>
+    <link href='https://unpkg.com/css.gg@2.0.0/icons/css/globe-alt.css' rel='stylesheet'>
+    <link href='https://unpkg.com/css.gg@2.0.0/icons/css/log-out.css' rel='stylesheet'>
+    <link href='https://unpkg.com/css.gg@2.0.0/icons/css/comment.css' rel='stylesheet'>
+    <link rel="stylesheet" type="text/css" href="/static/styles.css">
+    <script defer src="/static/script.js"></script>
+</head>
+<body>
+<div class="dropdown">
+    <i class="gg-globe-alt" id="globe-icon"></i>
+    <div class="dropdown-content" id="dropdown-content">
+        <a th:href="@{${httpServletRequest.requestURI} + '?page=' + ${currentPage} + '&lang=en'}">English</a>
+        <a th:href="@{${httpServletRequest.requestURI} + '?page=' + ${currentPage} + '&lang=sv'}">Svenska</a>
+    </div>
+</div>
+<div class="logout-link">
+    <a th:href="'/logout'">
+        <i class="gg-log-out"></i>
+    </a>
+</div>
+<div class="messages-link">
+    <a th:href="'/web/messages'">
+        <i class="gg-comment"></i>
+    </a>
+</div>
+<div id="profile-container">
+    <div id="img-container">
+        <img id="profile" th:src="${profilepic}" alt="No image found :/"/>
+        <div id="user-information">
+            <div id="username-container">
+                <p id="login" th:text="${userName}">default</p>
+            </div>
+            <div id="name-container">
+                <p id="name" th:text="${name}"></p>
+            </div>
+            <div id="email-container">
+                <p id="email" th:text="${email}">No email registered on user</p>
+            </div>
+            <a th:href="'/web/myprofile/edit'">
+                <button th:text="#{edit}"></button>
```

Comment:
There is something with the href that makes me able to press "Edit" even outside of the button. Like the space between button and the full width of profilecontainer or imagecontainer, on the right side of the button.

Reply:
Oh I see, I think I know how to fix this. Had this problem with some other buttons :)